chronograf.go

```diff
@@ -143,9 +143,9 @@ type TriggerValues struct {
 	Period        string `json:"period,omitempty"`         // Period length of time before deadman is alerted
 	Shift         string `json:"shift,omitempty"`          // Shift is the amount of time to look into the past for the alert to compare to the present
 	Operator      string `json:"operator,omitempty"`       // Operator for alert comparison
-	RangeOperator string `json:"range_operator,omitempty"` // RangeOperator is an optional operator for range comparisons
+	RangeOperator string `json:"rangeOperator,omitempty"` // RangeOperator is an optional operator for range comparisons
```

Comment:
Good catch about the go_fmt. Is the swagger.json hand-edited? I don't recall how to view and work with it.